### PR TITLE
Text analytics release notes Preview2

### PR DESCRIPTION
**Pull request by maririos**

### CODE REVIEW

File: releases/2020-02/dotnet.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ If you have a bug or feature request for one of the libraries, please [file an i
 
 Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
 
+### Text Analytics
+
+- Added the ability to create a TextAnalitycs client with a credential that can be updated in long-lived clients.
```
Comment (samvaity) [inline releases/2020-02/dotnet.md]: TextAnalitycs => TextAnalytics

File: releases/2020-02/dotnet.md [hunk 2]
```diff
@@ -56,6 +56,13 @@ If you have a bug or feature request for one of the libraries, please [file an i
 
 Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
 
+### Text Analytics
+
+- Added the ability to create a Text Analytics client with a credential that can be updated in long-lived clients.
+- The `TextAnalyticsError` model has been simplified and now collection objects include a `HasError` property that allows to check if an operation on a particular document succeeded or failed.
```
Comment (adrianhall) [inline releases/2020-02/dotnet.md]: This feels like two things, of which one is unimportant.  "Collection objects include a `HasError`...." is enough.

File: releases/2020-02/dotnet.md [hunk 3]
```diff
@@ -56,6 +56,13 @@ If you have a bug or feature request for one of the libraries, please [file an i
...
+- All batch overload methods have been renamed by adding the suffix `Batch` or `BatchAsync` accordingly.
+- All single text operation methods now return an atomic type of the operation result.
+- Various types have been renamed. See the Text Analytics [changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md#100-preview2-2020-02-11) for details.
```
Comment (adrianhall) [inline releases/2020-02/dotnet.md]: Move the change log to the header (### Text Analytics [Changelog]) and remove this one.

Comment (maririos) [reply releases/2020-02/dotnet.md]: Thanks!

**Pull request merged**